### PR TITLE
Followups to rasterise alg

### PR DESCRIPTION
**Pull request by nyalldawson**

### CODE REVIEW

File: python/plugins/processing/algs/qgis/Rasterize.py [hunk 1]
```diff
@@ -118,7 +112,7 @@ def initAlgorithm(self, config=None):
             QgsProcessingParameterNumber(
                 self.TILE_SIZE,
                 self.tr('Tile size'),
-                defaultValue=1024))
+                defaultValue=1024, minValue=0))
```
Comment (m-kuhn) [inline python/plugins/processing/algs/qgis/Rasterize.py]: Should `minValue` be 1? 0 will hang forever I guess.

**Review by m-kuhn — COMMENTED** [review]
Nice, thanks

File: python/plugins/processing/algs/qgis/Rasterize.py (hunk 1 above)
Comment (nyalldawson) [inline python/plugins/processing/algs/qgis/Rasterize.py]: Let's go with 64. I don't think there's a valid use case for 1 pixel wide tiles either ;)

Comment (nirvn) [inline python/plugins/processing/algs/qgis/Rasterize.py]: Tss tss tss, @nyalldawson , standing between the user and 1-pixel artform! 😉

Comment (m-kuhn) [inline python/plugins/processing/algs/qgis/Rasterize.py]: > Let's go with 64. I don't think there's a valid use case for 1 pixel wide tiles either ;)
  
Go for 1 and only allow setting powers of 2 ;)

File: python/plugins/processing/algs/qgis/CheckValidity.py [hunk 2]
```diff
@@ -158,7 +158,7 @@ def doCheck(self, method, parameters, context, feedback):
 
                     reason = "\n".join(reasons)
                     if len(reason) > 255:
-                        reason = reason[:252] + '...'
+                        reason = reason[:252] + '…'
```
Comment (m-kuhn) [inline python/plugins/processing/algs/qgis/CheckValidity.py]: Do you want to add that to spellchecker?

Comment (nyalldawson) [inline python/plugins/processing/algs/qgis/CheckValidity.py]: I tried once, add spell check went crazy. Guess it can't handle utf?

Comment (m-kuhn) [inline python/plugins/processing/algs/qgis/CheckValidity.py]: Ping @3nids

Comment (3nids) [inline python/plugins/processing/algs/qgis/CheckValidity.py]: I gave it a try https://github.com/qgis/QGIS/pull/5012/files
But there is so many in the code. I wonder if it makes sense to produce an error if someone writes ... in a comment.

**Pull request merged**